permissions/sailfish-office.profile

```diff
@@ -0,0 +1,13 @@
+# -*- mode: sh -*-
+
+# Firejail profile for /usr/bin/sailfish-office
+
+# x-sailjail-translation-catalog =
+# x-sailjail-translation-key-description =
+# x-sailjail-description = Execute sailfish-office application
+# x-sailjail-translation-key-long-description =
+# x-sailjail-long-description =
+
+### APPLICATION
+mkdir     ${HOME}/.local/share/Sailfish/Sailfish Office
+whitelist ${HOME}/.local/share/Sailfish/Sailfish Office
```

Comment:
Not gpg nor needed anymore?

Reply:
Oups, an old local modification that I commited by error with a `git commit -a`.

Reply:
Or worst, an old commit on top... Removing it.